target/linux/ramips/dts/mt7621_iptime_t5004.dts

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "iptime,t5004", "mediatek,mt7621-soc";
+	model = "ipTIME T5004";
+
+	aliases {
+		led-boot = &led_cpu;
+		led-failsafe = &led_cpu;
+		led-running = &led_cpu;
+		led-upgrade = &led_cpu;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_cpu: led-0 {
+			label = "blue:cpu";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_CPU;
```

Comment:
Ups, these need an extra 

 #include <dt-bindings/leds/common.h>

(I also noticed, the upstream example does not have a color. Still color are nice ;) )

Reply:
Oops, thanks. Never push without a build.